cli.js

```diff
@@ -12,12 +12,10 @@ meow({
 	]
 });
 
-reviewTimes(function (err, times) {
-	if (err) {
-		console.error(err.message);
-		process.exit(1);
-	}
-
+reviewTimes().then(function (times) {
 	console.log('iOS  ' + times.ios + ' days');
 	console.log('Mac  ' + times.mac + ' days');
+}, function (err) {
+	console.error(err.message);
+	process.exit(1);
 });
```

Comment:
Bump `meow` to 3.4.0 and you can drop this as `meow` handles uncaught rejected promises for us ;)